docs/google-cloud.md

```diff
@@ -34,107 +44,121 @@ gcloud auth configure-docker
 PATH="$PATH:$(pwd)/bin"
 ```
 
-### And run the configuration script
+### Run the configuration script
 
 ```
 kctf-config-create
 ```
 
-### Configuration properties
-Type a path for storing the challenges
+### Set configuration properties
+Enter a path for storing the challenges:
 ```
 ~/demo-ctf-cluster
```

Comment:
this needs to be "~/kctf-demo" for the command below to work

Reply:
done